docs/src/3.16FlowTypes.mdx

```diff
@@ -0,0 +1,38 @@
+import GithubLink from './utils/GithubLink'
+
+# Flow types
+
+Flow types are supported. To use the type, import directly from `regl-worldview`. More type definitions: <GithubLink href="https://github.com/cruise-automation/webviz/blob/master/packages/regl-worldview/src/types/index.js" />
```

Comment:
Also, instead of `More type definitions: <github icon link>` what about making the whole sentence "More type definitions." a link (maybe prefixed with the github icon)?

Reply:
Makes sense. Probably need more design refinement later
![image](https://user-images.githubusercontent.com/10999093/50406169-41526e00-0775-11e9-9686-4ef674a69d42.png)